docs/reference/indices/shrink-index.asciidoc

```diff
@@ -4,36 +4,30 @@
 <titleabbrev>Shrink index</titleabbrev>
 ++++
 
-The shrink index API allows you to shrink an existing index into a new index
-with fewer primary shards. The requested number of primary shards in the target index
-must be a factor of the number of shards in the source index. For example an index with
-`8` primary shards can be shrunk into `4`, `2` or `1` primary shards or an index
-with `15` primary shards can be shrunk into `5`, `3` or `1`. If the number
-of shards in the index is a prime number it can only be shrunk into a single
-primary shard. Before shrinking, a (primary or replica) copy of every shard
-in the index must be present on the same node.
+Shrinks an existing index into a new index with fewer primary shards.
 
-Shrinking works as follows:
 
-* First, it creates a new target index with the same definition as the source
-  index, but with a smaller number of primary shards.
+[source,console]
+----
+POST /twitter/_shrink/shrunk-twitter-index
+----
+// TEST[s/^/PUT twitter\n{"settings":{"index.number_of_shards":2,"blocks.write":true}}\n/]
 
-* Then it hard-links segments from the source index into the target index. (If
-  the file system doesn't support hard-linking, then all segments are copied
-  into the new index, which is a much more time consuming process. Also if using 
-  multiple data paths, shards on different data paths require a full copy of 
-  segment files if they are not on the same disk since hardlinks don’t work across
-  disks)
 
-* Finally, it recovers the target index as though it were a closed index which
-  had just been re-opened.
+[[shrink-index-api-request]]
+==== {api-request-title}
 
-[float]
-==== Preparing an index for shrinking
+`POST /<index>/_shrink/<target-index>`
+
+`PUT /<index>/_shrink/<target-index>`
+
+
+[[shrink-index-api-prereqs]]
+==== {api-prereq-title}
 
 In order to shrink an index, the index must be marked as read-only, and a
 (primary or replica) copy of every shard in the index must be relocated to the
-same node and have <<cluster-health,health>> `green`.
+same node and have a <<cluster-health, cluster health>> status of `green`.
```

Comment:
This reads awkardly--the subject is "a copy of every shard", but the last clause is referring to the node status. I'd probably make this a list:

Before you can shrink an index:

* The index must be read-only.
* A copy of every shard in the index must reside on the same node.
* The <<cluster-health, cluster health>> status must be `green`.

Reply:
Fixed with e6f9b25b736.